src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -78,4 +80,56 @@ public void testGroupByInQuery() {
     private JSONObject executeJdbcRequest(String query){
         return new JSONObject(executeQuery(query, "jdbc"));
     }
+
+    @Test
+    public void numberOperatorNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT ABS(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age IS NOT NULL LIMIT 5", "jdbc"),
+                executeQuery("SELECT abs(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age IS NOT NULL LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void trigFunctionNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT Cos(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age is NOT NULL LIMIT 5", "jdbc"),
+                executeQuery("SELECT cos(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age is NOT NULL LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void stringOperatorNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery(
+                        "SELECT SubStrinG(lastname, 0, 2) FROM elasticsearch-sql_test_index_account/account " +
+                                "ORDER BY age LIMIT 5", "jdbc"),
+                executeQuery(
+                        "SELECT substring(lastname, 0, 2) FROM elasticsearch-sql_test_index_account/account " +
+                                "ORDER BY age LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void binaryOperatorNameCaseInsensitiveTest() {
```

Comment:
I don't think `AND` binary operator is affected by this change. If that is true, this test is redundant.

Reply:
@abbashus Thanks for pointing out! 

Actually amaong all types of functions: 
number operators, trigonometric functions, string operators cannot go through the jdbc formatter;
binary operators and date functions can go through the jdbc formatter.

But I reckoned it would be good to test all five types of functions here over the case sensitivity, since no tests have ever tested even the binary operators and date functions. 
But was not quite sure about it, so do you think it would be better not to test them here?

Reply:
Rest what you tested accept `AND` are functions, but not `AND`. The parser is already case-insensitive to it. If `AND` actually would have been a function like `AND(a, b)`, it would have made sense. So, I personally feel it is not needed.

Reply:
@abbashus  You are right, I went back to the code and found the `AND`-like "binary operator" is no need to test since it has nothing with the function name case sensitivity that I was to fix. It was not "function" actually. I have deleted this block of code, thanks!

Reply:
@abbashus You are right, I went back to the code and found the AND-like "binary operator" is no need to test since it has nothing with the function name case sensitivity that I was to fix. It was not "function" actually. I have deleted this block of code, thanks!